### ISSUE TITLE
dotnet commands interpret string content

### ISSUE THREAD

**Issue by MangelMaxime** [Microsoft/dotnet#517]
Hello guys, not sure to be at the right place.

I am running this command:

```
dotnet pack -c Release /p:Version=1.0.0-beta-002 /p:PackageReleaseNotes="* BREAKING CHANGE: Fix icons display and implement Font Awesome helpers
 * Fix #61: Add `is-fullwidth` support for Table element
 * Fix #59: Add color support for the navbar compoenent
 * Fix #67: Enforce DSL synthax with `[<RequireQualifiedAccess>]` attribute
 * BREAKING CHANGE: Fix #70: Make button support different elements. To have the same behavior as before, replace: `Button.button` with `Button.button_a`"
```

And I get this error:

```
MSBUILD : error MSB1006: La propriété n'est pas valide.
Commutateur :  replace: `Button.button` with `Button.button_a`
```

Basically it's saying: `The property is invalid`

I suspect the engine, to catch the keyword `replace` even if it's in a string argument. I would expect the engine to not catch string content from cli arguments.

**Comment by MangelMaxime** [comment]
Ok, so it's seems the problems come form the `,`.

When I execute the next command, I removed the comma, it's working.
```
dotnet pack -c Release /p:Version=1.0.0-beta-003 /p:PackageReleaseNotes="* BREAKING CHANGE: Fix icons display and implement Font Awesome helpers
 * Fix #61: Add `is-fullwidth` support for Table element
 * Fix #59: Add color support for the navbar component
 * Fix #67: Enforce DSL synthax with `[<RequireQualifiedAccess>]` attribute
 * BREAKING CHANGE: Fix #70: Make button support different elements. To have the same behavior as before replace `Button.button` with `Button.button_a`"
```

**Comment by Petermarcu** [comment]
Glad you found the issue. http://github.com/nuget/home is probably the best place to open issues on dotnet pack. I'm going to go ahead and close this here but if you think there is a bug to be fixed, go ahead an open an issue there.
